src/main/java/com/maxmind/geoip2/model/AnonymousIpResponse.java

```diff
@@ -28,6 +28,10 @@ public AnonymousIpResponse(
             isResidentialProxy, isTorExitNode, network);
     }
 
+    /**
+     * Constructs an instance of {@code AnonymousIpResponse} based on the values in the passed
+     * in response.
+     */
```

Comment:
It looks like there is another constructor before this one that also needs a comment. Maybe the checkstyle check isn't working.

Also, should we document the parameters here and elsewhere? Can we enable a check for that?

Reply:
Did you make a change to the checkstyle config that will cause it to flag things like the missing constructor documentation above?

Reply:
The rule works and if you remove most of the constructors the check fails. This particular constructor was excluded due to `<property name="minLineCount" value="2"/>.` _ I removed it .

I( think it is a useful rule as these are mostly constructors that just call the super)

Reply:
With these constructors, it isn't a huge deal, but generally, I think we would want anything with parameters to be documented to make it clear what those parameters are. This seems independent of the line count.